pages/issuing/cardholder_terms.en.md

```diff
@@ -0,0 +1,176 @@
+
+Nevertheless, if a transaction exceeds the balance of the funds available on your Card, you shall remain fully liable to us for the amount of the transaction and any applicable fees.
+
+You do not have the right to stop payment on any purchase or payment transaction originated by use of your Card. If you authorize a transaction and then fail to make a purchase of that item as planned, the approval may result in a hold for that amount of funds for up to thirty (30) days. 
+
+### Returns and Refunds
+
+If you are entitled to a refund for any reason for goods or services obtained with your Card, you agree to accept credits to your Card for such refunds and agree to the refund policy of that merchant. The Issuer is not responsible for the delivery, quality, safety, legality or any other aspects of goods or services that you purchase from others with a Card. All such disputes must be addressed and handled directly with the merchant from whom those goods or services were provided.
+
+### Card Replacement
+
+If you need to replace your Card for any reason, please return to a branch of the financial institution where the Card was purchased to obtain an immediate replacement, or contact us at +855(0)23 532 0888 to request a replacement Card be mailed to you. You will be required to provide personal information which may include your Card number, full name, transaction history, etc. There is a fee for replacing your Card. Refer to the fee schedule provided within this document.
+
+### Card Transfers
+
+You may transfer funds to another card on the program.
+
+### Transactions Made In Foreign Currencies
+
+If you obtain your funds or make a purchase in a currency other than the currency in which your Card was issued, the amount deducted from your funds will be converted by BONGLOY into an amount in the currency of your Card. The exchange rate between the transaction currency and the billing currency used for processing international transactions is a rate selected by BONGLOY from the range of rates available in wholesale currency markets for the applicable central processing date, which may vary from the rate BONGLOY itself receives or the government-mandated rate in effect for the applicable central processing date. 
+
+### Receipts
+
+You should get a receipt at the time you make a transaction using your Card. You agree to retain, verify, and reconcile your transactions and receipts.
+
+### Card Account Balance/Periodic Statements
+
+You are responsible for keeping track of your Card Account available balance. Merchants generally will not be able to determine your available balance. It’s important to know your available balance before making any transaction. You may obtain information about the amount of money you have remaining in your Card Account by calling +855(0)23 532 0888 or viewing the available balance in the App.
```

Comment:
Remove the phone number here, we don't want people calling up asking what their balance is